staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/diskmanagers/vmdm.go

```diff
@@ -81,9 +81,9 @@ func (vmdisk vmDiskManager) Create(ctx context.Context, datastore *vclib.Datasto
 		}
 		if dsType != vclib.VSANDatastoreType {
 			klog.Errorf("The specified datastore: %q is not a VSAN datastore", datastore.Name())
-			return "", fmt.Errorf("The specified datastore: %q is not a VSAN datastore."+
-				" The policy parameters will work only with VSAN Datastore."+
-				" So, please specify a valid VSAN datastore in Storage class definition.", datastore.Name())
+			return "", fmt.Errorf("the specified datastore: %q is not a VSAN datastore."+
```

Comment:
Nit: Should we include this as a log message and then have a more concise error message? It seems a little strange to have a multi sentence error message.

Reply:
i prefer to not change anything other than casing in this pr to ease review.

the pr is making a number of small changes across a wide area of the code tree so getting all the reviews/approves will be hard enough and changing stuff makes this harder to review than needed.